mods/ra/weapons/explosions.yaml

```diff
@@ -142,15 +142,38 @@ CivPanicExplosion:
 		Delay: 1
 
 BarrelExplode:
+	Warhead@Cluster: FireCluster
+		Weapon: BarrelCluster
+		Dimensions: 3,3
+		Footprint: _X_ X_X _X_
```

Comment:
Can we change this to
```suggestion
		Footprint: _X_ XXX _X_
```
and then remove the other warheads from `BarrelExplode` and only use `BarrelCluster` (assuming we want to have a `LeaveSmudge` in the middle)?

Reply:
Looking at video of OG, the barrel cell has different effects than the surrounding four fireballs. I actually need the barrel itself to have a crater instead of scorch smudge. I also need the barrel explosion to be `building` explode instead of `large_napalm`. 

Will add the crater smudge tonight.